test/battle/ability/flash_fire.c

```diff
@@ -0,0 +1,26 @@
+#include "global.h"
+#include "test/battle.h"
+
+SINGLE_BATTLE_TEST("Flash Fire boosts fire type moves by 50% but no subsequent increase is applied")
```

Comment:
This test is not checking that the multiplier is kept at 1.5 the second time that Ember is used on Heatran, only that it increases once.